src/Controller/Base.php

```diff
@@ -424,8 +424,11 @@ protected function getContent($textQuery, $parameters = [], &$pager = [], $where
             return $this->storage()->getContent($textQuery, $parameters, $pager, $whereParameters);
         }
         $params = array_merge($parameters, $whereParameters);
+        unset($params['log_not_found']); // New storage system removes this functionality from the query engine
```

Comment:
1. Is this a BC Break?
2. Docs and config samples should be updated too.
3. I'm 99% sure that this param is also used somewhere else.

Reply:
> 3. I'm 99% sure that this param is also used somewhere else.

```
$ ag log_not_found
src/Controller/Frontend.php
152:        $content = $this->getContent($contenttype['slug'], ['slug' => $slug, 'returnsingle' => true, 'log_not_found' => !is_numeric($slug)]);

src/Legacy/Storage.php
1602:        if (isset($parameters['log_not_found'])) {
1603:            $logNotFound = $parameters['log_not_found'];
1604:            unset($parameters['log_not_found']);
```

In the case of the controller, that is only going to legacy, and legacy is legacy.

Reply:
Yes, this is only a legacy feature and I don't think the query parser is the right place for it. We have HTTP status exceptions and they can look after logging 404 errors if so needed. As with everything in this PR legacy stays exactly as is and we can decide for 4.0 onwards what we want to keep and drop. If in this testing phase a feature is found to be missing, then we will happily add it in, but there's no need to automatically include existing features if we think they don't fit.